cogboard-webapp/src/components/Widget/styled.js

```diff
@@ -51,6 +51,17 @@ StyledCard.propTypes = {
   theme: object.isRequired
 };
 
+export const ErrorMessage = (props) => {
+  const { errorMessage, errorCause } = props;
+
+  return (
+  <>
+    <Typography paragraph>{errorMessage}</Typography>
+    <Typography variant="caption" paragraph>{errorCause}</Typography>
+  </>
+  );
+};
+
```

Comment:
In my opinion, this component should be moved to a separate file. You may also extract props inside the parentheses. Like this `ErrorMessage = ({ errorMessage, errorCause }) => (<>...</>)`

Reply:
Each component should has it's own file. The exception to this are small styled components.

Reply:
done